src/docfx/lib/schema/JsonSchema.cs

```diff
@@ -11,6 +11,11 @@ namespace Microsoft.Docs.Build
     [JsonConverter(typeof(JsonSchemaConverter))]
     internal class JsonSchema
     {
+        /// <summary>
```

Comment:
nit: This is a custom extension, move it to line 193?

Reply:
OK.